### ISSUE TITLE
Setup reads previous values and makes them default

### ISSUE THREAD

**Issue by jjmartin** [awslabs/aws-lambda-redshift-loader#36]
This is more of a suggestion than an issue.   It would be nice for upgrades or just general changes if it would try to load any previously existing setup info and set that as the default for each of the questions.

**Comment by IanMeyers** [comment]
Agreed - happy to merge a pull which implements this, but wont be able to look at it for ~ 6 weeks.
